data/fields/religion.json

```diff
@@ -1,5 +1,31 @@
 {
     "key": "religion",
     "type": "combo",
-    "label": "Religion"
+    "label": "Religion",
+    "strings": {
+        "options": {
+            "bahai": "Bahá’í",
+            "benzhu": "Benzhu",
+            "buddhist": "Buddhist",
+            "caodaism": "Caodaist",
+            "chinese_folk": "Chinese Folk",
```

Comment:
This and “Vietnamese Folk” read a bit awkward. I wonder if the strings should be nouns rather than adjectives; “Chinese Folk Religion” sounds less awkward.

Reply:
Chinese Folk Religion seems far better but I am not a native speaker

Reply:
Maybe we can make an exception for the particular entries that don’t have a natural adjective form.